index.html

```diff
@@ -30,72 +30,108 @@ <h1>AMBROSE <br> ISIGBA</h1>
            highly invited to take a tour of my works.If you fall in 
             love with  my work gallery, and you are planning for one or 
            you need a code mentor. AMBROSE's CODELAB get you covered</p>
-        <button type="button">Start colaboration</button>
+        <button type="button">Start colaboration <img id="btn-imgA" src="./images/image_dots.png" alt=""></button>
         <div id="second-section-bg-img"></div>
   </main>
   <section id="project-sections">
      <h2>Projects</h2>
      <div id="project-container">
        <div class="full-section-div">
-         <div id="img-div1" class="imj-project-section"><img id="full-section-div1" src="./images/desktop7.png" alt=""></div>
+         <div id="img-div1" class="imj-project-section">
+           <img id="full-section-div1" src="./images/desktop7.png" alt="">
+           <img id="desktop-img1a" src="./images/image_dots.png" alt="">
+        </div>
+
          <div id="list-of-project1" class="list-of-project">
           <h4>Project names goes here</h4>
           <ul>
                <li class="project-list-1">HTML/CSS</li>
                <li class="project-list-2">Ruby on Rails</li>
                <li class="project-list-3">Javascricpt</li>
+               <li class="project-list-4">+1</li>
+
            </ul>
            <div class="project-link"><a href="#">See this project</a>
```

Comment:
- [ ] Your "See this project" button should be a button and behave like one(with hover, active styles etc). Wrapping it into a `<div>` is not the best practice to use a button. Kindly fix it and try to match it with the style that Figma template provides.
![image](https://user-images.githubusercontent.com/61919617/172934915-440f1c1e-dfc0-42c4-b7c9-79f6c258f3f1.png)
![image](https://user-images.githubusercontent.com/61919617/172934982-05cb522e-dbf1-4a8c-bdfd-f9f61d28abea.png)

Reply:
Done